### PR TITLE
bug fix scr_dead_marine

### PR DESCRIPTION
**Pull request by OH296**
fixes crash after loss of full fleet in fleet combat

### CODE REVIEW

File: ChapterMaster.yyp
Comment (GalacticChimp) [inline ChapterMaster.yyp:1]: Please update your GM version

Comment (OH296) [reply ChapterMaster.yyp:1]: roger that

**Comment by GalacticChimp** [comment]
seems to work, full fleet wipe didn't crash 

image

**Review by GalacticChimp — CHANGES_REQUESTED** [review]
fix works, please just update GM version and I'll merge the PR

**Comment by OH296** [comment]
alrighty version updated

**Review by GalacticChimp — APPROVED** [review]

**Pull request merged**